Makefile

```diff
@@ -79,7 +91,7 @@ help: ## Display this help.
 ##@ Development
 
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
Why is this changing from variables to concrete values?

Reply:
It seems new versions of the tool do not support these values, I thought this might have been in the migration docs, though I no longer see it.